examples/invocations/run-kritis-test.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: pipeline.knative.dev/v1beta1
+kind: TaskRun
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: unit-test-kritis-feature-x
+spec:
+    taskRef:
+        name: make
+    trigger:
+        triggerRef:
+            type: manual
+    inputs:
+        sources:
+            - name: 'kritis'
+              type: 'github'
+              url:  'github.com/grafeas/kritis'
+              branch: 'featureX'
+              commit: 'HEAD'
+        params:
+            - name: 'makeTarget'
+              type: 'string'
+              value: 'test'
+    results:
+        runs:
+        - name: 'runsBucket'
+          type: 'gcs'
+          url: 'gcs://somebucket/results/runs'
+        logs:
+        - name: 'logBucket'
+          type: 'gcs'
+          url: 'gcs://somebucket/results/logs'
+        tests:
+        - name: 'testBucket'
+          type: 'gcs'
+          url: 'gcs://somebucket/results/tests'
+status:
```

Comment:
hmm after a bit more thought and looking at @aaron-prindle's proposals around templating, I'm thinking the status section should also include the buildSpec with templating applied

Reply:
yes or a add a todo with a issue link describing templating proposal